src/api/result.c

```diff
@@ -68,9 +68,12 @@ int api_result(char *targetpluginkey, uint64_t callid,
   cinfo = connection_send_request(targetpluginkey, result, result_params,
       api_error);
 
+  if (true == api_error->isset)
```

Comment:
why not:

```c
if(api_error->isset) 
```

and

```c
if(!api_error->isset) 
```

instead of 

```c
if(true == api_error->isset) 
```

and 

```c
if(false == api_error->isset) 
```